apps/backend/src/features/api/api.service.ts

```diff
@@ -527,6 +528,24 @@ export class ApiService {
         }
     }
 
+    async deleteExternalComment({ deleteCommentDTO }: { deleteCommentDTO: DeleteCommentDTO }) {
+        const location = deleteCommentDTO.ownerLocation;
+        const url = `http://[${location}]/api/v2/posts/comment/delete`;
+        try {
+            return (
+                await axios.delete(url, {
+                    data: {
+                        postId: deleteCommentDTO.postId,
+                        commentId: deleteCommentDTO.commentId,
```

Comment:
instead of remapping all the objects, is it not possible to just pass data: { deleteCommentDto } ?

Reply:
Since keys of the body and the DTO are the same